enabler/src/de/schildbach/pte/AbstractNavitiaProvider.java

```diff
@@ -140,6 +140,17 @@ public AbstractNavitiaProvider(final NetworkId network, final String authorizati
 
     protected abstract String region();
 
+    private boolean preferNonNavitiaLineStyles = false;
+
+    /**
+     * Navitia is capable of returning "display_informations" which includes style related information. Depending on the
+     * coverage though, this may not be complete. Prefering non-Navitia line styles means we first delegate to
+     * {@link #lineStyle(String, Product, String)} before then falling back to {@link #getLineStyle(Product, String, String)}.
+     */
```

Comment:
Couldn't we try to automatically take what is provided in `display_informations` and fall back to `lineStyle()` automatically when it isn't available? So we wouldn't need this method and class variable?